presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -97,6 +97,50 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+To set up a Nessie instance locally using docker image, see `Setting up Nessie <https://projectnessie.org/try/docker/>`_. Once the docker instance is up and running, you should see logs similar to the following example:
+
+.. code-block:: none
+
+    2023-09-05 13:11:37,905 INFO  [io.quarkus] (main) nessie-quarkus 0.69.0 on JVM (powered by Quarkus 3.2.4.Final) started in 1.921s. Listening on: http://0.0.0.0:19120
+    2023-09-05 13:11:37,906 INFO  [io.quarkus] (main) Profile prod activated.
+    2023-09-05 13:11:37,906 INFO  [io.quarkus] (main) Installed features: [agroal, amazon-dynamodb, cassandra-client, cdi, google-cloud-bigtable, hibernate-validator, jdbc-postgresql, logging-sentry, micrometer, mongodb-client, narayana-jta, oidc, opentelemetry, reactive-routes, resteasy, resteasy-jackson, security, security-properties-file, smallrye-context-propagation, smallrye-health, smallrye-openapi, swagger-ui, vertx]
+
+
+If log messages related to Nessie's OpenTelemetry collector appear similar to the following example:
+
+.. code-block:: none
+
+    2023-08-27 11:10:02,492 INFO  [io.qua.htt.access-log] (executor-thread-1) 172.17.0.1 - - [27/Aug/2023:11:10:02 +0000] "GET /api/v1/config HTTP/1.1" 200 62
+    2023-08-27 11:10:05,007 SEVERE [io.ope.exp.int.grp.OkHttpGrpcExporter] (OkHttp http://localhost:4317/...) Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/127.0.0.1:4317
+
+you can disable OpenTelemetry using the configuration option ``quarkus.otel.sdk.disabled=true``.
```

Comment:
> I think it's better to move line 116 to be directly after line 109. Then after the code block, it's the "For example, start...."

@steveburnett which way you think is better?

Reply:
@yingsu00 great idea! I agree that moving line 116 ("you can disable OpenTelemetry") to directly follow line 109 ("If log messages related to Nessie's OpenTelemetry collector") is better, because it puts the solution earlier and makes it more visible to the reader, increasing the chance they will find it and solve their problem.